_includes/components/project-section.html

```diff
@@ -0,0 +1,21 @@
+<div class="project-section">
+  <div class="container">
+
+    <h2>{{ include.subhead }}</h2>
+
+    <p>{{ include.par1 }}</p>
+
+    {% if include.img1 != null %}
```

Comment:
change to `{% if include.img1 %}

Reply:
can you explain why

Reply:
and should this be done on every if function?

Reply:
basically include.img1 returns null if it doesn't exist so you don't have to check if its != to null.